### PR TITLE
fix: fixed and improved code example in readme

### PR DESCRIPTION
**Pull request by JohannesMerkt**
Signed-off-by: JohannesMerkt <mailjohannesmerkt@gmail.com>

### CODE REVIEW

**Review by relu91 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -133,25 +133,52 @@ the default servient.
 If you want to use node-wot as a library in your Node.js application, you can use npm to install the node-wot packages that you need. To do so, `cd` inside you application folder, and run:
 
 ```
-npm i @node-wot/core @node-wot/binding-coap --save
+npm i @node-wot/core @node-wot/binding-http --save
 ```
 
-Now, you can implement your node-wot entry point, e.g., `main.js` as follows:
+Now, you can implement your a thing as follows:
```
Comment (danielpeintner) [inline README.md:139]: either 
* "your thing" OR
* "a thing" without your

**Review by danielpeintner — CHANGES_REQUESTED** [review]
LGTM.

One minor typo that we should fix.

Thanks a lot!

File: README.md (hunk 1 above)
Comment (JohannesMerkt) [reply README.md]: Oh didnt see that. It should be fixed now. Thanks!

**Comment by danielpeintner** [comment]
Thanks, looks good. The test issues do not relate to this PR.

**Pull request merged**